### PR TITLE
main/zsh: upgrade to 5.4.2

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

File: main/zsh/APKBUILD [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
 pkgrel=0
 pkgdesc="A very advanced and programmable command interpreter (shell)"
 url="http://www.zsh.org/"
 arch="all"
 license="BSD"
+options="!check" # As of 5.4.2 - 3 tests fails
 makedepends="ncurses-dev"
 install="zsh.post-install zsh.post-upgrade zsh.pre-deinstall"
-source="http://www.zsh.org/pub/$pkgname-$pkgver.tar.xz
+source="http://www.zsh.org/pub/$pkgname-$pkgver.tar.gz
```
Comment (jbergstroem) [inline main/zsh/APKBUILD]: strange that `.xz` is missing upstream..

Comment (TBK) [reply main/zsh/APKBUILD]: I thought the same. Guess it is released by hand and somebody forgot.

Comment (jbergstroem) [reply main/zsh/APKBUILD]: I emailed the authors. One minor improvement for this abuild could be switching to https which seems to be working just fine.

Comment (TBK) [reply main/zsh/APKBUILD]: Ok. Which email address did you use? I need it to report the failing tests.

Comment (jbergstroem) [reply main/zsh/APKBUILD]: At the [main site](http://zsh.sourceforge.net) there's a maintainers link in the footer. Not sure if that's most accurate, but I thought it was worth a shot.

Comment (jbergstroem) [reply main/zsh/APKBUILD]: Sourceforge seems to have the `xz` tarball; don't know if its worth switching.

**Comment by TBK** [comment]
@jbergstroem I have changed the source to SF.net since it provices the xz package and offers a faster download.

**Comment by TBK** [comment]
The xz file on SF is giving abuild a stroke, so had to switch to gz.

**Comment by jirutka** [comment]
No, there's no cache enabled in Travis.

**Comment by jirutka** [comment]
Please don't change it to sourceforge, it's not confident and reliable source.

**Comment by TBK** [comment]
@jirutka K, after your previous comment I suspected that. I have just changed it back.

**Comment by jbergstroem** [comment]
Perhaps the tarballs are generated differently. The SHASUM at least seems to point at that.

**Pull request closed without merging**